README.md

```diff
@@ -146,9 +146,10 @@ By default svelte-themer persists the chosen theme with `localStorage`, and can
 
 `ThemeWrapper` will load a theme on first visit based on the following order:
 
+1. User-provided - The value specified in the `theme` prop.
 1. Saved - User's stored choice (from `localStorage`)
-2. Prefers - User's Operating System settings (via `prefers-color-scheme`)
-3. Fallback - First theme in `themes` specified (from presets, `light`)
+1. Prefers - User's Operating System settings (via `prefers-color-scheme`)
```

Comment:
this still orders properly in the markdown preview, but may we adjust to use 1,2,3,4?

Reply:
updated